lib/transaction/transaction.js

```diff
@@ -447,6 +447,8 @@ Transaction.prototype._fromNonP2SH = function(utxo) {
 };
 
 Transaction.prototype._fromMultisigUtxo = function(utxo, pubkeys, threshold) {
+  $.checkArgument(threshold <= pubkeys.length,
```

Comment:
Agreed with @braydonf on error message, and about the need of changing `address.js` to solve issue #1080.
To be sure, I'd add a test with @matiu's example, asserting it fails (see https://github.com/bitpay/bitcore/issues/1080)